EIPS/eip-4844.md

```diff
@@ -161,25 +162,29 @@ The execution layer verifies the wrapper validity against the inner `Transaction
 - The KZG commitments match the blob contents. (Note: this can be optimized with additional data, using a proof for a
   random evaluation at two points derived from the commitment and blob data)
 
-
-The signature is verified and `tx.origin` is calculated as follows:
+The hashes for signature verification and for determining the public transaction identifier are computed according to the [EIP-6493](./eip-6493.md) transaction signature scheme:
 
 ```python
-def unsigned_tx_hash(tx: SignedBlobTransaction) -> Bytes32:
-    # The pre-image is prefixed with the transaction-type to avoid hash collisions with other tx hashers and types
-    return keccak256(BLOB_TX_TYPE + ssz.serialize(tx.message))
+def compute_blob_sig_hash(message: BlobTransaction) -> Root:
+    domain = compute_transaction_domain(
+        BLOB_TX_TYPE,
+        BLOB_TX_TYPE_FORK_VERSION,
+        GENESIS_HASH,
+        CHAIN_ID,
+    )
+    return compute_signing_root(message, domain)
 
-def get_origin(tx: SignedBlobTransaction) -> Address:
-    sig = tx.signature
-    # v = int(y_parity) + 27, same as EIP-1559
-    return ecrecover(unsigned_tx_hash(tx), int(sig.y_parity)+27, sig.r, sig.s)
+def compute_blob_tx_hash(signed_tx: SignedBlobTransaction) -> Root:
```

Comment:
```suggestion
def tx_hash(signed_tx: SignedBlobTransaction) -> Root:
```
signed_tx_hash can now be referenced without ambiguity to be tx_hash

Reply:
```suggestion
def compute_tx_hash(signed_tx: SignedBlobTransaction) -> Root:
```